forest/shared/src/logger/mod.rs

```diff
@@ -2,15 +2,62 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use crate::cli::{cli_error_and_die, LogValue};
+use isatty::stdout_isatty;
 use log::LevelFilter;
+use pretty_env_logger::env_logger::WriteStyle;
 use std::str::FromStr;
 
-pub fn setup_logger(log_config: &[LogValue]) {
+#[derive(Debug)]
+pub enum LoggingColor {
+    Always,
+    Auto,
+    Never,
+}
+
+impl FromStr for LoggingColor {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "auto" => Ok(LoggingColor::Auto),
+            "always" => Ok(LoggingColor::Always),
+            "never" => Ok(LoggingColor::Never),
+            _ => Ok(LoggingColor::Auto),
```

Comment:
I'd expect `--color=cthulhu` to report an error not silently ignore the old gods. What do you think?

Reply:
You're right, fixed. Reports an error and errors out immediately